src/serlio/materials/MaterialUtils.cpp

```diff
@@ -193,4 +197,14 @@ std::filesystem::path getStingrayShaderPath() {
 	return sfxFile;
 }
 
+bool textureHasAlphaChannel(std::wstring path) {
+	const prt::AttributeMap* textureMetadata = prt::createTextureMetadata((FILE_PREFIX + path).c_str());
```

Comment:
mem leak! all return values of prt functions starting with `createXYZ` need manual releasing (through smart ptr of course)

Reply:
fixed.